pkg/clients/sprayproxy/sprayproxy.go

```diff
@@ -105,3 +102,10 @@ func getPaCHost() (string, error) {
 	}
 	return fmt.Sprintf("https://%s", route.Spec.Host), nil
 }
+
+func buildBodyData(pacHost string) []byte {
+	data := make(map[string]string)
+	data["url"] = pacHost
+	bytesData, _ := json.Marshal(data)
```

Comment:
In case of an error while marshaling, shouldn't we log it?

Reply:
fixed.